Sources/Sentry/SentryScreenshotIntegration.m

```diff
@@ -10,6 +11,23 @@
 #import "SentrySDK+Private.h"
 
 #if SENTRY_HAS_UIKIT
+
+void
+saveScreenShot(const char *path)
+{
+    NSString *reportPath = [NSString stringWithUTF8String:path];
+    if (![NSFileManager.defaultManager fileExistsAtPath:reportPath isDirectory:nil]) {
```

Comment:
`h`: What if there are already files at the specified path? Are we going to override them in `[SentryDependencyContainer.sharedInstance.screenshot saveScreenShots]`? I think we could delete them here.